File: SWIFT/Cap04/c04ex07.swift
// c04ex07.swift

import Foundation

var numero: Int

print("Entre um numero inteiro: ", terminator: "")
guard let input = readLine(), let numero = Int(input) else {
    fatalError("Entrada invalida.")
}

if (numero >= 20 && numero <= 90) {
    print("O numero esta na faixa de 20 a 90.")
} else {
    print("O numero esta fora da faixa de 20 a 90.")
}

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: KOTLIN/Cap04/c04ex07.kt
// c04ex07.kt

fun main() {
    var Numero: Int = 0

    print("Entre um numero inteiro: ")
    Numero = readLine()?.toInt() ?: 0

    if (Numero >= 20 && Numero <= 90) {
        println("O numero esta na faixa de 20 a 90.")
    } else {
        println("O numero esta fora da faixa de 20 a 90.")
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
